src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/XmlDsigBase64Transform.cs

```diff
@@ -66,7 +66,7 @@ public override void LoadInput(object obj)
 
         private void LoadStreamInput(Stream inputStream)
         {
-            if (inputStream == null) throw new ArgumentException("obj");
+            if (inputStream == null) throw new ArgumentException(nameof(inputStream));
```

Comment:
This should be reverted.  The argument to the public API is `obj`.
```suggestion
            if (inputStream == null) throw new ArgumentException("obj");
```
(though in either case this is being passed as the message rather than the parameter name)